javascript/multiBracketValidation/README.md

```diff
@@ -0,0 +1,23 @@
+# Code Challenge: Multi Bracket Validation
+
+## CC=Code Challenge/Pair Programming
+
+Tina Myers
+
+## Languages:
+
+javascript
+
+## Approach/Efficiency:
+
+## API:
```

Comment:
For this challenge you don't need this API section here, that's more for when you implement a data structure and are creating methods on it